index.js

```diff
@@ -58,6 +70,58 @@ function ClassList (node) {
     attribute = tokens.join(' ')
   }
 
+  function forEach (callback, thisArg) {
+    tokens.forEach(callback, thisArg)
+  }
+
+  function entries () {
+    return {
+      [Symbol.iterator]: () => {
+        let index = 0
+        return {
+          next: () => {
+            if (index < tokens.length) {
+              return { value: [index, tokens[index++]], done: false }
+            }
+            return { done: true }
+          }
+        }
+      }
+    }
+  }
+
+  function keys () {
+    return {
```

Comment:
Same as entries comment: `return tokens.keys()`

Reply:
Oh, it seems to be so. I didn't notice that.

Reply:
so it just only improve `toggle`🤣